tests/drivers/spi/spi_loopback/boards/atsame54_xpro.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_0"
```

Comment:
Why is this change here?

Reply:
So you can run the `spi_loopback` test on this board - other boards [have it too](https://github.com/zephyrproject-rtos/zephyr/tree/master/tests/drivers/spi/spi_loopback/boards/)

Reply:
Sure, I was wondering "why part of this PR" - is the SPI driver already merged?

Reply:
Alright I moved it to the SPI PR.